Traceback (most recent call last):
  File "c:\users\sk250164\onedrive - teradata\documents\gitlab\venv\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\users\sk250164\onedrive - teradata\documents\gitlab\venv\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\users\sk250164\onedrive - teradata\documents\gitlab\venv\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\users\sk250164\onedrive - teradata\documents\gitlab\venv\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\sk250164\AppData\Local\Continuum\anaconda3\lib\asyncio\base_events.py", line 584, in run_until_complete
    return future.result()
  File "c:\users\sk250164\onedrive - teradata\documents\gitlab\venv\lib\site-packages\nbclient\client.py", line 664, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\users\sk250164\onedrive - teradata\documents\gitlab\venv\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "c:\users\sk250164\onedrive - teradata\documents\gitlab\venv\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import spacy
nlp=spacy.load('en_core_web_sm')

nlp.pipe_names
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[1;32m~\AppData\Local\Temp\ipykernel_1728\1352668536.py[0m in [0;36m<module>[1;34m[0m
[1;32m----> 1[1;33m [1;32mimport[0m [0mspacy[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      2[0m [0mnlp[0m[1;33m=[0m[0mspacy[0m[1;33m.[0m[0mload[0m[1;33m([0m[1;34m'en_core_web_sm'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      3[0m [1;33m[0m[0m
[0;32m      4[0m [0mnlp[0m[1;33m.[0m[0mpipe_names[0m[1;33m[0m[1;33m[0m[0m

[1;31mModuleNotFoundError[0m: No module named 'spacy'
ModuleNotFoundError: No module named 'spacy'

